app/models/user.rb

```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
+  has_secure_password
+
+  validates :username, :password, :email, presence: true
+  validates :username, :email, uniqueness: true
+end
```

Comment:
do we want to include first_name and last_name in the user table? @CjMoore @meyerhoferc @valerietrudell